.github/ISSUE_TEMPLATE/al-runtime-bug.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Business Central SaaS/On-Prem Runtime Bug
+about: 'Report a bug in the behavior of the cloud/on-prem product. This includes SQL errors, API issues, runtime errors and incorrect behavior of AL code during execution.'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+🔻🔻🔻This repository is only dedicated to bug reports for AL compiler and developer tools. Runtime issues should follow the support process indicated below. 🔻🔻🔻
+
+# Official Support Processes
+- Open a support request to CSS through the PartnerSource portal or the Business Central Admin Center.
```

Comment:
Hyperlinks
```suggestion
- Open a support request to CSS through the [PartnerSource portal](https://partner.microsoft.com/support) or the [Business Central Admin Center](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/tenant-admin-center#internal-administrators).
```

Reply:
I added links to our documentation instead because these may change